pkg/config/constants/constants.go

```diff
@@ -153,9 +153,10 @@ const (
 
 	WaypointServiceAccount = "istio.io/for-service-account"
 
-	ManagedGatewayLabel          = "gateway.istio.io/managed"
-	ManagedGatewayController     = "istio.io-gateway-controller"
-	ManagedGatewayMeshController = "istio.io-mesh-controller"
+	ManagedGatewayLabel               = "gateway.istio.io/managed"
+	ManagedGatewayController          = "istio.io-gateway-controller"
+	ManagedGatewayMeshControllerLabel = "istio.io-mesh-controller"
+	ManagedGatewayMeshController      = "istio.io/mesh-controller"
```

Comment:
In general, these constants should have comments, otherwise they are ambiguous to discern.

Reply:
make sense, i think these may even change